docs/docs/core/hooks/use-fetch-client.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: useFetchClient
+slug: /hooks/use-fetch-client
+description: API reference for the useFetchClient hook in Strapi
+tags:
+  - hooks
+  - fetch-client
+---
+
+## Usage
+
+The following examples shows a basic way to use the `useFetchClient` hook to make a get request to a strapi backend endpoint:
+
+```jsx
+import {useState} from "react"
+import useFetchClient from '@strapi/admin/admin/src/hooks/useFetchClient';
+
+const Component () => {
+    const [items, setItems] = useState([]);
+    const { get } = useFetchClient();
+    const requestURL = "/some-endpoint";
+
+    const handleGetData = () => {
+        const { data } = await get(requestURL);
+        setItems(data.items);
+    }
+
+    return(
+        <div>
+            <div>
+                {
+                    items && items.map(item => <h1>{item.name}</h1>)
+                }
+            </div>
+            <button onClick={handleGetData}>Show Items</button>
+        </div>
+    )
+}
+```
+
+## Methods
+
+Essentially, this is an abstraction around the axios instance wrapped in a hook. It provides a simple interface to handle API calls to the Strapi backend.
+It handles request cancellation under the hook with [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController), and when the component gets unmounted all the reqeusts that it has made are aborted.
+
+The hook exposes four methods:
+
+- **get(url, config)**: which is a method that given a relative url (string), will make a GET request to the Strapi backend with the configuration (object) passed as second parameter, if needed.
+- **post(url, data, config)**: which is a method that given a relative url (string), will make a POST request with the data (object) provided to the Strapi backend with the configuration passed as third parameter (object), if needed.
+- **put(url, data, config)**: which is a method that given a relative url (string), will make a PUT request with the data (object) provided to the Strapi backend with the configuration passed as third parameter (object), if needed.
+- **del(url, config)**: which is a method that given a relative url (string), will make a DELETE request to the Strapi backend with the configuration passed as second parameter (object), if needed.
+
+## Implementation details
+
+The fetch instance is running axios under the hook with two request interceptors.
+
+### Base URL
+
+The default URL will be the one defined in the environment variable: `STRAPI_ADMIN_BACKEND_URL`.
+
+### Interceptors
+
+#### Request
+
+The request interceptor adds the following parameters to the header:
+
+```js
+{
+    Authorization: `Bearer <AUTH_TOKEN>`,
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+}
+```
+
+#### Response
+
+If everything has gone correctly, the response is returned as it comes from the backend. However, if it cointains a **status code of 401**, the authentication details will be removed from
```

Comment:
```suggestion
If everything has works correctly, the response is returned as it comes from the backend. However, if it contains a **status code of 401** the authentication details will be removed from
```

Reply:
I guess here we should remove "has" before "works"